# Req
string gz_partition
string parameters
---
# Result
bool success
